project/Dependencies.scala

```diff
@@ -39,4 +41,7 @@ object Dependencies {
   val mavenPluginApi = "org.apache.maven" % "maven-plugin-api" % mavenApiVersion
   val mavenPluginAnnotations = "org.apache.maven.plugin-tools" % "maven-plugin-annotations" % mavenAnnotationsVersion % Provided
   val mavenScalaPlugin = "net.alchim31.maven" % "scala-maven-plugin" % mavenScalaPluginVersion
+
+  val ipcsocket = "org.scala-sbt.ipcsocket" % "ipcsocket" % ipcsocketVersion
+  val monix = "io.monix" %% "monix" % monixVersion
```

Comment:
Thanks for adding that here, but I think you forgot to update `build.sbt` afterwards?

Reply:
LOL, yes.